xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/Makefile.in

```diff
@@ -3,11 +3,21 @@ SRCS=
 ifeq (@USE_OPENGL@,1)
 SRCS += RendererVAAPI.cpp
 SRCS += RendererVDPAU.cpp
+SRCS += RendererStagefright.cpp
```

Comment:
I was not sure if android-x86 was opengl or gles. If the latter i they need to go here - yep.